pkgs/development/compilers/vala/default.nix

```diff
@@ -15,19 +15,7 @@ let
     # header file isn't available at all, but that patch (./gvc-compat.patch)
     # can be shared between all versions of Vala so far.
     graphvizPatch =
-      let
-        fp = { commit, sha256 }: fetchpatch {
```

Comment:
Maybe this is only needed for vala 0.38 and 0.42?

Reply:
The goal was to avoid having to maintain the patch ourselves. But in the end we had to because openembedded was too slow.

Reply:
Seems that the patch has been removed in https://github.com/openembedded/openembedded-core/commit/dc4555d3cb6109e57c2dcfcc6713f69e1ef99755. So maybe we have to maintain ourselves if we don't switch to `--disable-valadoc`?